tests/topotests/bgp_redistribute_table/r2/bgpd.conf

```diff
@@ -0,0 +1,11 @@
+debug bgp zebra
```

Comment:
Please disable this, we agreed on disabling debug logging for CI.

Reply:
yes